packages/@react-spectrum/switch/stories/Switch.stories.tsx

```diff
@@ -10,91 +10,104 @@
  * governing permissions and limitations under the License.
  */
 
-import {action} from '@storybook/addon-actions';
-import React from 'react';
-import {storiesOf} from '@storybook/react';
+import {chain} from '@react-aria/utils';
+import {ComponentMeta, ComponentStoryObj} from '@storybook/react';
+import React, {useState} from 'react';
 import {Switch} from '../';
 
-storiesOf('Switch', module)
-  .addParameters({providerSwitcher: {status: 'positive'}})
-  .add(
-    'Default',
-    () => render()
-  )
-  .add(
-    'defaultSelected: true',
-    () => render({defaultSelected: true})
-  )
-  .add(
-    'isSelected: true',
-    () => render({isSelected: true})
-  )
-  .add(
-    'isSelected: false',
-    () => render({isSelected: false})
-  )
-  .add(
-    'isDisabled: true',
-    () => render({isDisabled: true})
-  )
-  .add(
-    'isEmphasized: true',
-    () => render({isEmphasized: true})
-  )
-  .add(
```

Comment:
i have removed this story entirely, it's covered in chromatic and it wouldn't have behavioral differences from the regular disabled story